website/docs/components/table/partials/code/component-api.md

```diff
@@ -41,9 +53,85 @@ The Table component itself is where most of the options will be applied. However
     If set, determines the vertical alignment for all cell (`td`) content in a table. Does not apply to table headers (`th`).
   </C.Property>
   <C.Property @name="caption" @type="string">
-    Adds a (non-visible) caption for users with assistive technology. If set on a sortable table, the provided table caption is paired with the automatically generated sorted message text.
+    Adds a (non-visible) caption for users with assistive technology. If set on a sortable table, the provided table caption is paired with the automatically generated sorted message text. Note: it is best practice to provide a table caption for users with assistive technology.
   </C.Property>
   <C.Property @name="...attributes">
     Supported for the `Hds::Table` component.
   </C.Property>
+  <C.Property @name="onSort" @type="function">
+    Callback function that is invoked when one of the sortable table headers is clicked (or has a keyboard interaction performed). The function receives the values of `sortBy` and `sortOrder` as arguments.
+  </C.Property>
+</Doc::ComponentApi>
+
+### Table::Tr
+
+This component is not eligible to receive interactions (e.g., it cannot have an `onClick` event handler attached directly to it). Instead, an interactive element should be placed _inside_ of the `Th`, `ThSort`, or `Td` elements.
+
+It can contain `Hds::Table::Th`, `Hds::Table::ThSort`, or `Hds::Table::Td` components.
+
+<Doc::ComponentApi as |C|>
+    <C.Property @name="...attributes">
+    This component supports the use of `...attributes`.
+  </C.Property>
+</Doc::ComponentApi>
+
+### Table::Th
+
+This component is not eligible to receive interactions (e.g., it cannot have an `onClick` event handler attached directly to it). Instead, an interactive element should be placed _inside_ of the `Th` element.
+
+If the `Th` component is passed as the first "cell" of a table body row, it has `scope="row"` automatically applied to it for accessibility purposes.
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="align" @type="enum" @values={{array "left" "center" "right" }} @default="left">
+    If set, determines the text alignment.
+  </C.Property>
+  <C.Property @name="scope" @type="string" @values={{array "col" "row" }} @default="col">
+    If used as the first item in a table body's row, `scope` should be set to `row` for accessibility purposes. Note: you only need to manually set this if you're creating a custom table using the child components, instead of the standard invocation.
```

Comment:
```suggestion
    If used as the first item in a table body’s row, `scope` should be set to `row` for accessibility purposes. Note: you only need to manually set this if you’re creating a custom table using the child components, instead of the standard invocation.
```